### PR TITLE
🌟[feat]: Use D3 / C3 for chats

### PR DESCRIPTION
**Pull request by yshing**
Use C3 / D3 for performance.

### CODE REVIEW

**Comment by cswbrian** [comment]
image
The gender chart disappeared, while male should be on left and female on right

image
For desktop, can you make it a row with 3 columns, instead of 3 rows with one column?

**Comment by cswbrian** [comment]
image
Can you sort the chart by descending order?

File: package.json [hunk 1]
```diff
@@ -8,7 +8,9 @@
     "@material-ui/core": "^4.8.3",
     "@material-ui/icons": "^4.5.1",
     "@material-ui/lab": "^4.0.0-alpha.40",
+    "c3": "^0.7.12",
     "csvtojson": "^2.0.10",
+    "d3": "^5.15.0",
```
Comment (cswbrian) [inline package.json:13]: Should we remove `"plotly.js` from package.json as well?

File: src/components/organisms/ConfirmedCaseVisual.js [hunk 2]
```diff
@@ -86,47 +86,9 @@ export default function ConfirmedCaseVisual(props) {
   )
 
   const isMobile = useMediaQuery({ maxWidth: 960 })
```
Comment (cswbrian) [inline src/components/organisms/ConfirmedCaseVisual.js:88]: Is it possible to use `${bps.down("md")}`?

**Comment by cswbrian** [comment]
@nandiheath is the `yarn.lock` required?

File: package.json (hunk 1 above)
Comment (yshing) [reply package.json:13]: Sure, but that doesn't really affect on performance.
And the yarn.lock changes on that is quite large

**Comment by cswbrian** [comment]
image
What does the '0' mean?

image
Can the english also use our main style?

**Comment by cswbrian** [comment]
Everything good!

File: src/components/organisms/ConfirmedCaseVisual.js (hunk 2 above)
Comment (cswbrian) [reply src/components/organisms/ConfirmedCaseVisual.js:88]: @yshing Except this.

**Review by cswbrian — CHANGES_REQUESTED** [review]

File: src/components/organisms/ConfirmedCaseVisual.js (hunk 2 above)
Comment (cswbrian) [reply src/components/organisms/ConfirmedCaseVisual.js:88]: I will skip this first 🐱

Comment (yshing) [reply src/components/organisms/ConfirmedCaseVisual.js:88]: @cswbrian No, It's not CSS it's inside JS but we can 
```suggestion
  const isMobile = useMediaQuery({ maxWidth: bps.values.md })
```

**Pull request merged**